addons/web/static/tests/helpers/mock_server.js

```diff
@@ -373,17 +401,19 @@ var MockServer = Class.extend({
      * @private
      * @param {string} model a model name
      * @param {any[]} domain
+     * @param {Object} param2
```

Comment:
param2?

Reply:
Param is not named due to destructuring, what do you suggest as doc name?

Reply:
why not simply params?

Reply:
In the hypothetical case of having 2 such destructed objects as param, there needs to be something to differentiate them, so I just suffixed it with its position among other params (0 indexed).

But ok to use just `param` in this case.